src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -176,10 +173,23 @@ class WorkspaceInvitePage extends React.Component {
 
     updateOptionsWithSearchTerm(searchTerm = '') {
         const {personalDetails, userToInvite} = OptionsListUtils.getMemberInviteOptions(this.props.personalDetails, this.props.betas, searchTerm, this.getExcludedUsers());
+
+        // update selectedOptions as well
```

Comment:
Can you please update this comment to explain _why_ we are doing this?

Reply:
Like `We need to update selectedOptions as well when personalDetails or policyMembers changes`?

Reply:
Something like `We need to update selectedOptions as well when policyMembers or personalDetails props changes`?

Reply:
Updated